src/github.com/getlantern/lantern-mobile/app/src/main/AndroidManifest.xml

```diff
@@ -27,7 +27,7 @@
             android:theme="@style/FinestWebViewTheme.Light" />
 
         <activity
-            android:name=".activity.LanternMainActivity"
+            android:name=".activity.LanternMainActivity_"
```

Comment:
A requirement when using Android annotations: you have to rename your activities with an underscore appended at the end. See here for more info: https://github.com/excilys/androidannotations/wiki/HowItWorks